src/ape/api/query.py

```diff
@@ -102,7 +99,7 @@ def estimate_query(self, query: QueryType) -> Optional[int]:
         query engine is not available for use or is unable to complete the query.
 
         Args:
-            query (``QueryType``): Query to estimate execution time for.
+            query (``QueryType``): Query for which execution time will be estimated.
```

Comment:
`will be estimated` is passive-language. Consider:

```suggestion
            query (``QueryType``): Query to estimate.
```